dagger/golang/options.go

```diff
@@ -0,0 +1,55 @@
+package golang
+
+import "github.com/caarlos0/env/v6"
+
+type config struct {
+	GoBaseImage string   `env:"BASE_IMAGE,notEmpty" envDefault:"docker.io/golang"`
```

Comment:
```suggestion
	GoBaseImageRepository string   `env:"BASE_IMAGE,notEmpty" envDefault:"docker.io/golang"`
```

Reply:
I will keep the existing var name for consistency between other packages. I used same name in multiple places